.travis.yml

```diff
@@ -1,47 +1,72 @@
-
+sudo: false
+os:
+  - linux
+  - osx
```

Comment:
It's possible that they've only made one mega-image for `osx` with *everything* in it, but on `linux`, when `language` is `cpp`, you either have just one or no JVM at all, for example, while the `java` one has a bunch, and the `go` setting gives you a bunch of different golang versions, `cpp` has just one, etc...

You could make any of them compile C++ or do the rest by installing more packages, but since I'm no Java fiend and wasn't going to test over all the available JVMs, I picked the option that looked the smallest and install what it was missing, hoping it'd help with booting up faster. :wink: